api/src/pcapi/core/offers/repository.py

```diff
@@ -351,6 +353,10 @@ def get_collective_offers_by_filters(
             )
         q2 = subquery.subquery()
         query = query.join(q2, q2.c.collectiveOfferId == educational_models.CollectiveOffer.id)
+    if formats is not None:
```

Comment:
Suggestion : j'imagine que tu préférerais éviter un `overlap([])` :
```suggestion
    if formats:
```